### PR TITLE
Improved key api.

### PR DESCRIPTION
**Pull request by geekbeast**

### CODE REVIEW

File: src/main/java/com/kryptnostic/v2/storage/api/KeyStorageApi.java [hunk 1]
```diff
@@ -58,19 +60,19 @@
     Response setRSAPublicKey( @Body byte[] publicKey );
 
     @GET( CONTROLLER + RSA_PUBLIC_KEY_PATH + USER_ID_PATH )
-    byte[] getRSAPublicKey( @Path( USER ) UUID user );
+    byte[] getRSAPublicKey( @Path( USER ) UUID user);
```
Comment (UnsungHero97) [inline src/main/java/com/kryptnostic/v2/storage/api/KeyStorageApi.java]: most of these changes are just whitespace

File: src/main/java/com/kryptnostic/v2/storage/api/KeyStorageApi.java [hunk 2]
```diff
@@ -96,16 +104,32 @@ Response setObjectCryptoService(
...
     @POST( CONTROLLER + VERSIONED_CRYPTO_SERVICES_PATH )
     Map<VersionedObjectKey, byte[]> getObjectCryptoServices( @Body VersionedObjectKeySet ids );
 
     /**
+     * Returns AES encrypted crypto services.
+     * @param ids
+     * @return
+     */
+    @POST( CONTROLLER + AES_CRYPTO_SERVICES_PATH )
+    Map<VersionedObjectKey, BlockCiphertext> getAesEncryptedCryptoServices( @Body VersionedObjectKeySet ids );
```
Comment (UnsungHero97) [inline src/main/java/com/kryptnostic/v2/storage/api/KeyStorageApi.java]: can we have the same API but only with object ids, and auto-assume latest version?

File: src/main/java/com/kryptnostic/v2/storage/api/KeyStorageApi.java [hunk 3]
```diff
@@ -96,16 +104,32 @@ Response setObjectCryptoService(
      * @param version The version of the object for which to retrieve a crpyto service
      * @return The crypto service corresponding to the object with version specified by {@code version} and object id
      */
+    @Deprecated
     @GET( CONTROLLER + CRYPTO_SERVICE_PATH + OBJECT_ID_PATH + VERSION_PATH )
-    byte[] getObjectCryptoService( @Path( OBJECT_ID ) UUID objectId, @Path( VERSION ) long version );
+    byte[] getObjectCryptoService( @Path( OBJECT_ID ) UUID objectId, @Path( VERSION ) long version);
```
Comment (UnsungHero97) [inline src/main/java/com/kryptnostic/v2/storage/api/KeyStorageApi.java]: can we have the same API but only with object ids, and auto-assume latest version?

File: src/main/java/com/kryptnostic/v2/storage/models/LoadLevel.java [hunk 4]
```diff
@@ -3,5 +3,6 @@
 public enum LoadLevel {
     ID,
     METADATA,
+    CRYPTO,
```
Comment (UnsungHero97) [inline src/main/java/com/kryptnostic/v2/storage/models/LoadLevel.java]: ooooooooooh :)

File: src/main/java/com/kryptnostic/v2/storage/api/KeyStorageApi.java (hunk 3 above)
Comment (geekbeast) [inline src/main/java/com/kryptnostic/v2/storage/api/KeyStorageApi.java]: This one is being deprecated.

File: src/main/java/com/kryptnostic/v2/storage/api/KeyStorageApi.java (hunk 2 above)
Comment (geekbeast) [inline src/main/java/com/kryptnostic/v2/storage/api/KeyStorageApi.java]: Yes.

**Pull request merged**